pegasus_wrapper/workflow.py

```diff
@@ -90,7 +90,7 @@ def from_params(params: Parameters) -> "WorkflowBuilder":
             default_site=params.string("site"),
             conda_script_generator=CondaJobScriptGenerator.from_parameters(params),
             namespace=params.string("namespace"),
-            default_resource_request=ResourceRequest.from_parameters(params),
+            default_resource_request=ResourceRequest.from_parameters(params.namespace("slurm")),
```

Comment:
You could argue for making such a change, but it will break a lot of existing stuff and should be discussed, so please don't make the change on this PR and make a new issue if you would like to propose it.